cli/src/etos_client/downloader/downloader.py

```diff
@@ -142,6 +107,49 @@ def __save_file(self, item: Downloadable, response: requests.Response) -> None:
         with open(download_path, "wb+") as report:
             for chunk in response:
                 report.write(chunk)
+        if not self.__verify(item, download_path):
+            self.logger.error("Checksum verification failed on %r", item)
```

Comment:
Will there only be this log message? Shouldn't it propagate further so that the client can exit with exit code > 0?